oak_containers_stage1/src/main.rs

```diff
@@ -146,3 +171,17 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     image::switch(&args.init).context("error switching to the system image")?
 }
+
+/// Tries to parse a string slice as an address.
+///
+/// Assumes that the address is a hexadecimal representation if it starts with "0x", or decimal
+/// otherwise.
+fn try_parse_phys_addr(arg: &str) -> anyhow::Result<PhysAddr> {
```

Comment:
I'm surprised there's not an easier way to do this :(

Reply:
Yes, I couldn't find a better way.